### PR TITLE
Croesus profit overlay

### PR DESCRIPTION
**Pull request by efefury**

### CODE REVIEW

File: src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/DungeonNpcProfitOverlay.java [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+		String lastOpenChestName = SBInfo.getInstance().lastOpenChestName;
+		Matcher matcher = pattern.matcher(lastOpenChestName);
+		if (!matcher.matches()) return;
+
+		GuiChest guiChest = (GuiChest) event.gui;
+		if (guiChest.inventorySlots.getSlot(10).getStack() == last) {
+			render(guiChest);
+			return;
+		}
+		System.out.println("further");
```
Comment (romangraef) [inline src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/DungeonNpcProfitOverlay.java:89]: me when further

Comment (efefury) [reply src/main/java/io/github/moulberry/notenoughupdates/miscfeatures/DungeonNpcProfitOverlay.java:89]: yes it's not finished yet lulonaut will be *remaking* it

**Review by hannibal002 — APPROVED** [review]

**Review by NopoTheGamer — CHANGES_REQUESTED** [review]
image
it does this like 40% of the time

**Review by NopoTheGamer — APPROVED** [review]

**Review by romangraef — APPROVED** [review]

**Review by hannibal002 — APPROVED** [review]
System.out.println("further");

**Pull request merged**